docs/data-sources/artifactory_file.md

```diff
@@ -21,6 +21,10 @@ The following arguments are supported:
 * `path` - (Required) The path to the file within the repository.
 * `output_path` - (Required) The local path the file should be downloaded to.
 * `force_overwrite` - (Optional) If set to true, an existing file in the output_path will be overwritten. Default: false
+*  `download_latest_artifact` - (Optional) If set to `true`, HCL will send the artifact path directly to Artifactory without verification 
+   if the file exists. Used in maven repositories, where handling releases and snapshots enabled. For example,
```

Comment:
I don't like a maven specific flag for a generic resource. Are there any other repo types that do this? I don't want a param that conditionally applies - that's the reason we have resource specific repos